packages/playwright/tests/test_utils.js

```diff
@@ -164,7 +164,8 @@ export async function login(page) {
 
   // Take a screenshot of header that should have "Create feature" button.
   // console.log('take a screenshot of header that should have "Create feature" button');
-  await expect(page.getByTestId('header')).toHaveScreenshot('after-login-click.png');
+  // This is no longer very useful.
```

Comment:
Is there a situation where a developer should uncomment this?  If not, can we just delete this line?

Reply:
I guess not.  Easy to add again if we want.